src/main/java/auth/BasicAuth.java

```diff
@@ -29,4 +29,9 @@ public HttpHeaders getAuthHeaders() {
             }
         };
     }
+
+    @Override
+    public String toString(){
+        return "BasicAuth [username=" + username + ", password=" + password + "]";
```

Comment:
would this object every get `toString`'d and sent somewhere where it shouldn't? like email or something? if so we should remove `password` from this